### PR TITLE
Finishes off the tape refactor

### PR DESCRIPTION
**Pull request by AsciiSquid**
## About The Pull Request
Phases out all ingame inclusions of sticky tape with the new stacked tape. Along with a few minor bug fixes regarding said new tape.

## Why It's Good For The Game
It's been a long time coming to be honest.

## Changelog
:cl:
fix: corrected stacking errors regarding duct tapes
refactor: refactored /obj/item/stack/sticky_tape to the more generic /obj/item/stack/tape
/:cl:

### CODE REVIEW

File: whitesands/code/game/objects/items/stacks/tape.dm [hunk 1]
```diff
@@ -18,6 +18,11 @@
 	var/list/conferred_embed = EMBED_HARMLESS
 	var/overwrite_existing = FALSE
 
+/obj/item/stack/tape/merge(obj/item/stack/S) //Because we have unique childern, we need to add an additional fail case
```
Comment (triplezeta) [inline whitesands/code/game/objects/items/stacks/tape.dm]: childern

Comment (AsciiSquid) [reply whitesands/code/game/objects/items/stacks/tape.dm]: Childern

File: code/_globalvars/lists/maintenance_loot.dm [hunk 2]
```diff
@@ -107,6 +107,8 @@ GLOBAL_LIST_INIT(common_loot, list( //common: basic items
 		/obj/item/stack/sheet/metal/twenty = 1,
 		/obj/item/stack/sheet/mineral/plasma = 1,
 		/obj/item/sign = 1,
+		/obj/item/stack/tape/industrial,
+		/obj/item/stack/tape/industrial/electrical,
```
Comment (ZephyrTFA) [inline code/_globalvars/lists/maintenance_loot.dm]:
```suggestion
		/obj/item/stack/tape/industrial = 1,
		/obj/item/stack/tape/industrial/electrical = 1,
```

File: whitesands/code/game/objects/items/stacks/tape.dm [hunk 3]
```diff
@@ -18,6 +18,11 @@
 	var/list/conferred_embed = EMBED_HARMLESS
 	var/overwrite_existing = FALSE
 
+/obj/item/stack/tape/merge(obj/item/stack/S) //Because we have unique children, we need to add an additional fail case
+	if(src.type != S.type)
+		return
+	. = ..()
```
Comment (ZephyrTFA) [inline whitesands/code/game/objects/items/stacks/tape.dm]:
```suggestion
	return ..()
```

**Pull request merged**